### ISSUE TITLE
LateInitUsage annotation exclusion

### ISSUE THREAD

**Issue by vanniktech** [arturbosch/detekt#241]
When I have a property like this:

```kotlin
@Inject @field:SchedulerMainThread lateinit var mainScheduler: Scheduler
```

and I set it up to ignore `@Inject` it'll still get flagged since `@SchedulerMainThread` is still present.

I'd like to change the way the exclusion work, meaning if one value matches it should be ignored.

I can't find a case where the current behavior (needing to exclude both) makes more sense.

**Comment by Mauin** [comment]
I agree. As long as one of the exclude strings matches at least one annotation on the `lateinit var` we should not report that property.
